assets/views/home/references.ejs

```diff
@@ -3,7 +3,9 @@
     <tr>
       <td class="firstHeading" colspan="2"><h3>Trades ({{numberOfTrades()}})</h3></td>
       <td><h3 class="text-center"><small></small></h3></td>
-      <td><h3 ng-if="user.isFlairMod && !onIndexPage"><small>Approved</small></h3></td>
+      <<td><h3 ng-if="user.isFlairMod && !onIndexPage"><small>Approved: </small></h3>
+ +        <span ng-if="user.isFlairMod && !onIndexPage">{{refUser.references.filter(isTrade).filter(isApproved).length}}</span>
+ +    </td>
```

Comment:
Looks like these lines have `+`'s on them that shouldn't be there (did you copy+paste from the diff?) so they're actually showing up on the page:

<img width="117" alt="screen shot 2017-10-17 at 17 04 20" src="https://user-images.githubusercontent.com/8943230/31695100-3df3d20c-b35d-11e7-9ea0-593b180224b9.png">